File: day-5/array-sum.c
#include <stdio.h>
#include <stdlib.h>
int main (void){
    int a[100],limit,i;
    printf("Enter your array limit :");
    scanf("%d",&limit);
    printf("Enter your values :");
    for ( i = 0; i < limit; i++)
    {
        scanf("%d",&a[i]);
    }
    int sum=0;
    for ( i = 0; i < limit; i++)
    {
        sum=sum+a[i];
    }
    printf("Sum of the Array :%d",sum);
    




 







    return EXIT_SUCCESS;
}

File: day-7/ArraySum.java
import java.util.Scanner;

public class ArraySum {
    public static void main(String[] args) {
        System.out.println("Enter your limit for the Array :");
        try (Scanner s = new Scanner(System.in)) {
            int limit=s.nextInt();
            System.out.print("Enter values for the array");
            int[] arr = new int[limit];

            for ( int i = 0; i < limit; i++) {
                 arr[i]=s.nextInt();
                
                
            }
            //sum
            int sum=0;
            for (int i = 0; i < limit; i++) {
                sum=arr[i]+sum;
                
            }
            System.out.print("Sum of array :"+sum);
        }
    }
    
}
